diff --git a/cmake/AddBipedalLocomotionUnitTest.cmake b/cmake/AddBipedalLocomotionUnitTest.cmake
index 82430fd..4449f25 100644
--- a/cmake/AddBipedalLocomotionUnitTest.cmake
+++ b/cmake/AddBipedalLocomotionUnitTest.cmake
@@ -69,6 +69,7 @@ function(add_bipedal_test)
       target_link_libraries(${targetname} PRIVATE CatchTestMain ${${prefix}_LINKS})
       target_compile_definitions(${targetname} PRIVATE CATCH_CONFIG_FAST_COMPILE CATCH_CONFIG_DISABLE_MATCHERS)
       target_compile_features(${targetname} PUBLIC cxx_std_17)
+      target_compile_definitions(${targetname} PRIVATE -D_USE_MATH_DEFINES)
 
       add_test(NAME ${targetname} COMMAND ${targetname})
 
